### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by Quannichan**
Refactor: add btn hamburger menu

### CODE REVIEW

File: src/components/footer_form.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<<<<<<< Updated upstream
```
Comment (Lanng21) [inline src/components/footer_form.html]: Pls double check code

File: src/components/portfolio.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<<<<<<< Updated upstream
+=======
```
Comment (Lanng21) [inline src/components/portfolio.html]: Same

**Comment by Quannichan** [comment]
Review my code, please 😭

File: src/components/portfolio.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                        <div class="port_img5" id="img5_phone" alt="img5"></div>
+                    </div>
+                    <div class="wrap_img">
+                        <div class="port_img3" id="img3_phone" alt="img3"></div>
+                    </div>
+                </div>
+            </div>
+            <div class="wrap_page_nav">
+                <img class="nav_arr" id="left" src="../images/arrow.png" alt="nav">
+                    <a class="choose_nav" href="">1</a>
```
Comment (Lanng21) [inline src/components/portfolio.html]: can use list?

File: src/components/portfolio.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                    <a class="choose_nav" href="">3</a>
+                    <a class="choose_nav" href="">4</a>
+                    <a class="choose_nav" href="">5</a>
+                <img class="nav_arr" id="right" src="../images/arrow.png" alt="nav">
+            </div>
+        </div>
+    </div>
+    </div>
+</section>
+>>>>>>> Stashed changes
```
Comment (Lanng21) [inline src/components/portfolio.html]: check

File: src/stylesheet/layouts/large_screen/_footer_large.scss [hunk 5]
```diff
@@ -0,0 +1,122 @@
+<<<<<<< Updated upstream
```
Comment (Lanng21) [inline src/stylesheet/layouts/large_screen/_footer_large.scss]: check

**Review by Lanng21 — COMMENTED** [review]
pls double check all file code and resolve conflict

**Review by Lanng21 — APPROVED** [review]

**Pull request merged**